test/cqlpy/test_tools.py

```diff
@@ -1737,6 +1737,22 @@ def test_scylla_sstable_query_null_data(cql, test_keyspace, scylla_path, scylla_
     with scylla_sstable(simple_clustering_table, cql, test_keyspace, scylla_data_dir) as (table, schema_file, sstables):
         args = [scylla_path, "sstable", "query", "--output-format", "json", "--scylla-yaml-file", f"{os.path.dirname(scylla_data_dir)}/conf/scylla.yaml", "--logger-log-level", "scylla-sstable=debug"]
         args.extend(sstables)
+
+
+@pytest.mark.parametrize("tag", [
+        "sstable-dump-4d1cc6a4-6c13-11f0-a3c9",
+        "test-4d1cc6a46c1311f0a3c9"
+])
+def test_scylla_sstable_query_data_from_snapshot(cql, test_keyspace, scylla_path, scylla_data_dir, tag):
+    with scylla_sstable(simple_clustering_table, cql, test_keyspace, scylla_data_dir) as (table, schema_file, sstables):
+        host = cql.cluster.contact_points[0]
+        args = [scylla_path, "nodetool", "snapshot", "--host", host, "--tag", tag, "--keyspaces", test_keyspace]
+        try:
+            subprocess.check_output(args, stderr=subprocess.STDOUT)
+        except subprocess.CalledProcessError as e:
+            pytest.fail(f"Failed to create snapshot: {e}\n{e.output.decode('utf-8')}")
+        args = [scylla_path, "sstable", "query", "--output-format", "json", "--scylla-yaml-file", f"{os.path.dirname(scylla_data_dir)}/conf/scylla.yaml", "--logger-log-level", "scylla-sstable=debug"]
+        args.extend([f"{os.path.dirname(sst)}/snapshots/{tag}/{os.path.basename(sst)}" for sst in sstables])
```

Comment:
Please consider adding a "finally" that del_snapshot the snapshot you created.
Without that, what happens if you try to run this test twice (e.g., test/cqlpy/run --count=2 test_tools.py::test_scylla_sstable_dump_data_from_snapshot) - it will try to snapshot twice to the same name, does that work?

Reply:
This option doesn't seem to work anymore for `test/cqlpy/run`.  Tried with ./test.py --repeat=2 and there is not problem since each run uses a random keyspace and table names, but I'm ok with deleting the snapshot in any case.

Reply:
Weird, I'm running from the scylla dbuild docker container.
```
$ export SCYLLA=build/debug/scylla; test/cqlpy/run -v --count 2 test_tools.py::test_scylla_sstable_query_data_from_snapshot

Scylla is: /home/bhalevy/dev/scylla/build/debug/scylla.
Booting Scylla on 127.1.160.174 in /tmp/scylla-test-41134...
Boot successful (1.7 seconds).
Driver name ScyllaDB Python Driver, version 3.28.2
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --count
  inifile: /home/bhalevy/dev/scylla/test/pytest.ini
  rootdir: /home/bhalevy/dev/scylla/test
```

Reply:
Maybe dbuild doesn't have pytest-repeat installed - as I said it's needed for dtest (which can't run in dbuild as far as I remember), but once I installed it, I liked it and kept it, it's a generally useful plugin.

Reply:
pytest-repeat wasn't added to the frozen toolchain yet.